packages/components/build/config-tagged-commit.mjs

```diff
@@ -0,0 +1,91 @@
+/*
```

Comment:
how does this config differ from the one used to build components normally? I.e. the config.mjs here. Is there a reason we can't just reuse that config here?

Reply:
It has different paths. In order to avoid duplicating a config file we can modify `node-build` so that it, instead of a config file name, will be able to to take a config object. Then we can load a config file in a tagged commit, modify paths, and provide an updated config file to the `Builder` class. @balmas, what do you think about a solution like that?

That would open some other interesting possibilities because we could have many more build parameters defined in a config object (file) than we do now. It will make things much more flexible for future extension.

Reply:
I think that's okay, but let's postpone it until after the release since it isn't essential.

Reply:
OK. The reason I decided to have two copy of config files because `alpheios-core` is the only place where we need that. It is not required neither for `embed-lib`, nor for `webextension`.

I will enter an issue for that.

Reply:
thanks!